packages/cli/src/commands/seed.ts

```diff
@@ -0,0 +1,59 @@
+import {Command} from '@oclif/command'
+import {join} from 'path'
+import pkgDir from 'pkg-dir'
+import {log} from '@blitzjs/display'
+import {runMigrate} from './db'
+
+const projectRoot = pkgDir.sync() || process.cwd()
+
+const seedFunctions = ['default', 'seed']
```

Comment:
we should just support a default exported function, i don't think supporting `export const seed =` is necessary. this should also simplify a bunch of the code in `run` as well